arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -73,6 +73,18 @@ using namespace arangodb::application_features;
 using namespace arangodb::basics;
 using namespace arangodb::options;
 
+namespace {
+arangodb::CreateDatabaseInfo createExpressionVocbaseInfo(arangodb::application_features::ApplicationServer& server) {
+  arangodb::CreateDatabaseInfo info(server, arangodb::ExecContext::current());
+  auto rv = info.load("Z", std::numeric_limits<uint64_t>::max());
```

Comment:
Why is this database named "Z"?
I guess it has to do with sort order, but there could still be other databases sorted after "Z"...? :thinking:

Reply:
Name is actually does not matter.  Database even is not added to database list. So sorting is also not relevant. I`ll add a comment.